contributors/devel/contributor-cheatsheet.md

```diff
@@ -0,0 +1,39 @@
+# Kubernetes
+
+| Repo | PRs | Issues | Notes |
+| ---- | --- | ------ | ----- |
+| [k/kubernetes](https://github.com/kubernetes/kubernetes) | [PRs](https://github.com/kubernetes/kubernetes/pulls) | [Issues](https://github.com/kubernetes/kubernetes/issues) | [Meeting Notes](http://bit.ly/kubenotes)
+| [k/community](https://github.com/kubernetes/community) | [PRs](https://github.com/kubernetes/community/pulls) | [Issues](https://github.com/kubernetes/community/issues) |
+| [k/docs](https://github.com/kubernetes/kubernetes.github.io) | [PRs](https://github.com/kubernetes/kubernetes.github.io/pulls) | [Issues](https://github.com/kubernetes/kubernetes.github.io/issues)
+
+## Workflow
+
+- [Gubernator Dashboard - k8s.reviews](https://k8s-gubernator.appspot.com/pr)
+- [reviewable.kubernetes.io](https://reviewable.kubernetes.io/reviews#-)
```

Comment:
Change it to "Reviewable for Kubernetes" to be consistent with rest of the page? Another nit, if you like the suggestion.